config/_default/menus/menus.en.yaml

```diff
@@ -0,0 +1,1263 @@
+  - name: APM
+    identifier: "agent_docker_apm"
+    url: agent/docker/apm/
+    parent: "agent_docker"
+    weight: 201
+  - name: Log collection
+    identifier: "agent_docker_log"
+    url: agent/docker/log/
+    parent: "agent_docker"
+    weight: 203
+
+  ## AGENT - Kubernetes
+
+  - name: "Kubernetes"
+    url: "agent/kubernetes/"
+    parent: "agent"
+    identifier: "agent_kubernetes"
+    weight: 3
+  - name: "DaemonSet Setup"
+    url: "agent/kubernetes/daemonset_setup/"
+    parent: "agent_kubernetes"
+    weight: 301
+  - name: "Host Setup"
+    url: "agent/kubernetes/host_setup/"
+    parent: "agent_kubernetes"
+    weight: 302
+  - name: "Cluster Agent"
+    url: "agent/kubernetes/cluster/"
+    parent: "agent_kubernetes"
+    weight: 303
+  - name: "Using Helm"
+    url: "agent/kubernetes/helm/"
+    parent: "agent_kubernetes"
+    weight: 304
+  - name: "Event Collection"
+    url: "agent/kubernetes/event_collection/"
+    parent: "agent_kubernetes"
+    weight: 305
+  - name: "Custom Integrations"
+    url: "agent/kubernetes/integrations/"
+    parent: "agent_kubernetes"
+    weight: 306
+  - name: "Metrics"
+    url: "agent/kubernetes/metrics/"
+    parent: "agent_kubernetes"
+    weight: 307
+  - name: "Custom Metrics & DogStatsD"
+    url: "agent/kubernetes/dogstatsd/"
+    parent: "agent_kubernetes"
+    weight: 308
+  - name: "Legacy Versions"
+    url: "agent/kubernetes/legacy/"
+    parent: "agent_kubernetes"
+    weight: 309
+
+  ## AGENT - Autodiscovery
+
+  - name: "Autodiscovery"
+    url: "agent/autodiscovery/"
+    identifier: "agent_autodiscovery"
+    parent: "agent"
+    weight: 4
+  - name: "Integrations Templates"
+    url: "agent/autodiscovery/integrations/"
+    parent: "agent_autodiscovery"
+    weight: 401
+  - name: "Container Identifier"
+    url: "agent/autodiscovery/ad_identifiers/"
+    parent: "agent_autodiscovery"
+    weight: 402
+  - name: "Template Variables"
+    url: "agent/autodiscovery/template_variables/"
+    parent: "agent_autodiscovery"
+    weight: 403
+  - name: "Tag Extraction"
+    url: "agent/autodiscovery/tag/"
+    parent: "agent_autodiscovery"
+    weight: 404
+  - name: "Discovery Management"
+    url: "agent/autodiscovery/management/"
+    parent: "agent_autodiscovery"
+    weight: 405
+  - name: "Troubleshooting"
+    url: "agent/autodiscovery/troubleshooting/"
+    parent: "agent_autodiscovery"
+    weight: 406
+  - name: "Cluster Checks"
+    url: "agent/autodiscovery/clusterchecks/"
+    parent: "agent_autodiscovery"
+    weight: 407
+
+  ## AGENT - LOGS
+
+  - name: "Log Collection"
+    url: "agent/logs/"
+    identifier: "agent_logs"
+    parent: "agent"
+    weight: 5
+  - name: "Advanced Log Collection"
+    url: "agent/logs/advanced_log_collection"
+    parent: "agent_logs"
+    weight: 501
+  - name: "Proxy"
+    url: "agent/logs/proxy"
+    parent: "agent_logs"
+    weight: 502
+
+  - name: "Proxy"
+    url: "agent/proxy/"
+    parent: "agent"
+    weight: 6
```

Comment:
Proxy doesn't show in the nav. Check the preview.

Reply:
fixed